### ISSUE TITLE
Proxy has problem with some situations 

### ISSUE THREAD

**Issue by HamidRezaAshkiyan** [SadeghHayeri/GreenTunnel#77]
**Describe the bug**
When Green Tunnel Starts for me it unblock YouTube very well but when i want to download from YouTube with IDM i can not. and also i cannot connect to green tunnel at port 8000 or even when i click use system proxy. 

Can you tell me why this is happening? Is this even a bug?

Side Question: when i download from inside country does it calculate half? 
And it would be a great idea if we can use direct connection for sites inside country. 
I know i talked a lot for just one issue!

**Comment by SadeghHayeri** [comment]
Hi,
for using green-tunnel with IDM, I think you should check HTTP proxy settings for IDM (127.0.0.1:8000)

and for the second question, if your question is about Iran policies, half price it's based on your requests destination and it is only for some websites and not all of the `.ir` websites, you can check the list [here](https://g2b.ito.gov.ir/index.php/site/page/view/list_ip).
(and yes for websites that are on the list, I think the price is calculated half, but what's the point? non of them are blocked)

**Comment by HamidRezaAshkiyan** [comment]
The only reason i said that is I hate to disconnect and connect the app ( VPN ) I want to have both benefit of unblocking sites beside having direct connection to sites inside country. So that was the reason for my question.

**Comment by HamidRezaAshkiyan** [comment]
![image](https://user-images.githubusercontent.com/38791873/88623034-4dbd4c00-d0b9-11ea-9aa2-d0a9c9693be7.png)

![image](https://user-images.githubusercontent.com/38791873/88623107-7fceae00-d0b9-11ea-91a7-9475ce715c6a.png)
Note: green tunnel is connect! its red and i reported in another issue. #76  

as you can see it works on youtube but my IDM and telegram doesnt connect ( not even by use system setting or set manual proxy to localhost) 

Do you have same problem?

**Comment by SadeghHayeri** [comment]
Humm, I think GreenTunnel has a bug on your system, you can run green-tunnel CLI for more info (currently GUI has no any log outputs) or disable proxy settings on IDM.
also, GreenTunnel can't use it with Telegram, because the regime blocks Telegram using IP addresses and there is no way to reach the servers from Iran.
